homeassistant/components/light/hue.py

```diff
@@ -179,6 +179,7 @@ def update_lights():
                 new_lights.append(lights[light_id])
             else:
                 lights[light_id].info = info
+                lights[light_id].update_ha_state()
```

Comment:
I'm not sure about this. It seems to work fine, but I'm not sure if there any side-effects.